src/render/object.rs

```diff
@@ -103,6 +103,12 @@ impl InstanceDesc {
     }
 }
 
+impl Default for InstanceDesc {
```

Comment:
do we still need these?

Reply:
Dunno. Could always revert them and silence the warning if you prefer.

Reply:
Oh, I see now that this is about `Default`. I don't think these defaults do anything useful. If you don't mind, let's disable the warning on missing defaults - it's somewhat controversial.